ceph.spec.in

```diff
@@ -248,6 +248,12 @@ on commodity hardware and delivers object, block and file system storage.
 Summary:       Ceph Base Package
 Group:         System Environment/Base
 Requires:      ceph-common = %{epoch}:%{version}-%{release}
+# redhat-lsb-core/lsb-release for lsb_release executable
+%if 0{?suse_version}
+Requires:      lsb-release
+%else
+Requires:      redhat-lsb-core
```

Comment:
@ktdreyer Grepping for `lsb_release` shows that this executable is getting run in lots more places than just `lsb_release_parse()`. Would you be willing to accept a patch that adds a runtime dependency on the `lsb-release` package for SUSE and Debian (and derivatives)? This would fix the bug on those distros and would be a nice stopgap until all instances of `lsb_release` can be excised from the codebase.